README.md

```diff
@@ -125,7 +125,11 @@ For example:
 config :my_app, MyApp.SlackBot,
   app_token: "MY_SLACK_APP_TOKEN",
   bot_token: "MY_SLACK_BOT_TOKEN",
-  bot: MyApp.SlackBot
+  bot: MyApp.SlackBot,
+# add this if you want to access channels other than public
+  channels: [
+    types: ["public_channel", "im", "private_channel"]
+  ]
```

Comment:
By default it will join all conversations it is part of/allowed to:

https://github.com/ryanwinchester/slack_elixir/blob/main/lib/slack/channel_server.ex#L7-L11

Reply:
@ryanwinchester it won't work for `private_channel` unless you add this explicitly.

Reply:
yeah, but what I'm saying is, that by default, we add _all_ channel types. This config option is for if you _**don't**_ want that.